codeguru_profiler_agent/utils/time.py

```diff
@@ -3,11 +3,14 @@
 import time
 from datetime import datetime
 
+
 def to_iso(epoch_milli):
     try:
-        return datetime.fromtimestamp(epoch_milli / 1000).isoformat()
+        return datetime.utcfromtimestamp(epoch_milli / 1000).isoformat(
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The naive datetime objects are treated by many datetime methods as local times, it is preferred to use aware datetimes to represent times in UTC. The recommended way to create an aware datetime object representing a specific timestamp in UTC is by using `now` or `fromtimestamp` methods and passing `tzinfo` to them.

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)